src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -140,33 +150,63 @@ If you need to install the agent manually, follow these steps:
   <Step>
     ### (Android-only) Configure your Android app [#configure-android]
 
-    If you have an Android-native app, you'll need make the following changes in your Android app:
+    If you have an Android-native app, you'll need make the following changes:
+
+   1. Add the following changes to apply the Gradle plugin:
+
+    If your project is using Plugins DSL (Flutter 3.16 or Later), make the following changes:
```

Comment:
I think we prefer `higher` instead of `later`. This is what we have in [our style guide](https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#version).